Actions/DetermineArtifactUrl/DetermineArtifactUrl.ps1

```diff
@@ -17,16 +17,9 @@ try {
     #region Action: Determine artifacts to use
     $telemetryScope = CreateScope -eventId 'DO0084' -parentTelemetryScopeJson $parentTelemetryScopeJson
 
-    $insiderSasToken = ""
```

Comment:
In _BuildALGoProject.yaml there's
```
env:
    Secrets: '${{ steps.ReadSecrets.outputs.Secrets }}'
```
when calling _DetermineArtifactUrl_ action. 
It should be removed.

Reply:
Good catch